File: app/br/ufc/insightlab/vonqbe/model/ReturnListFilesUpload.scala
package br.ufc.insightlab.vonqbe.model

import play.api.libs.json.{Format, Json}

class ReturnListFilesUpload(var name : String, var file1 : UploadFileResponse, var file2 : UploadFileResponse) {

  override def toString: String = "FILES ------------\n"+
    "name : " +name +"\n"+
    "file1 : " +file1+"\n"+
    "file2 : " +file2

  implicit val format : Format[ReturnListFilesUpload] = Json.format[ReturnListFilesUpload]

}

File: src/main/java/br/ufc/insightlab/vonqbe/model/ReturnListFilesUpload.java
package br.ufc.insightlab.vonqbe.model;

public class ReturnListFilesUpload {
	
	public String name;
	
	public UploadFileResponse file1;
	
	public UploadFileResponse file2;
	

	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public UploadFileResponse getFile1() {
		return file1;
	}
	public void setFile1(UploadFileResponse file1) {
		this.file1 = file1;
	}
	public UploadFileResponse getFile2() {
		return file2;
	}
	public void setFile2(UploadFileResponse file2) {
		this.file2 = file2;
	}
	

}
